src/codechandler/TTMLCodecHandler.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "../parser/TTML.h"

```

Comment:
do you think is possible replace all these "..." with a full path reference?

Reply:
What I think is better here is to move TTML.h into the codechandler folder as it's only used by this file anyway.

Reply:
for me ok, but IMO also better under a `/codechandler/ttml/` folder,
which will have to be dropped in the future...,
i had started to think/create the TTML parser on Kodi core as done with WebVTT,
but it is really complex format with tons of features, which have to be compared with the limitations of libass renderer,
W3C define allow extensive customisation of implementation but i am still in difficulty on how to create a first parser version

Reply:
Sounds good. I've moved into ttml folder and fixed formatting. Should we do a cleanup on this or leave?

Reply:
not strictly necessary as they will have to be deleted
i could do it in a separate PR i would like to make a refactor of the helper class in to different files
i will think of something i will write to you on slack if not today next days